docs/guide/advance-format.md

```diff
@@ -0,0 +1,66 @@
+---
+id: advance-format
+title: 🐛 Advance Format (Msgpack)
+description: >-
+  Learn how to use MessagePack (MsgPack) for efficient binary serialization in Fiber applications.
+sidebar_position: 9
+---
+
+# 🐛 Advance Format
+
+## Msgpack
+
+Fiber enables efficient binary serialization using MessagePack (MsgPack). You can leverage popular Go libraries to encode and decode MsgPack data within your route handlers.
+
+- Fiber supports binding requests with the `application/vnd.msgpack` content type by default. For more details, see the [Binding documentation](../api/bind.md#msgpack).
+- Use `Ctx.MsgPack()` to bind MsgPack data directly to structs, similar to how you would use JSON binding. Alternatively, use `Ctx.AutoFormat()` to send response as MsgPack when the Accept HTTP header is `application/vnd.msgpack`, For more details, see the [AutoFormat documentation](../api/ctx.md#autoformat).
+
+### Recommended Libraries
+
+- [github.com/vmihailenco/msgpack](https://pkg.go.dev/github.com/vmihailenco/msgpack) — A widely used, feature-rich MsgPack library.
+- [github.com/shamaton/msgpack/v2](https://pkg.go.dev/github.com/shamaton/msgpack/v2) — High-performance MsgPack library.
+
+### Installation
+
+Install either library using:
+
+```bash
+go get github.com/vmihailenco/msgpack
+# or
+go get github.com/shamaton/msgpack/v2
+```
+
+> **Note:** Fiber does **not** register MsgPack by default because it is not part of the Go standard library. You can choose from several popular MsgPack libraries in the Go ecosystem. The two recommended packages below are widely used and compatible with Go.
```

Comment:
You can give a reference to this line in msgpack section of binder and ctx pages

Reply:
Solved